lib/features/append/AppendKeyboardBindings.js

```diff
@@ -0,0 +1,74 @@
+import inherits from 'inherits-browser';
+
+import KeyboardBindings from 'diagram-js/lib/features/keyboard/KeyboardBindings';
+
+
+/**
+ * BPMN 2.0 specific keyboard bindings.
+ *
+ * @param {Injector} injector
+ */
+export default function AppendNodeKeyboardBindings(injector) {
+
+  this._injector = injector;
+  this._keyboard = this._injector.get('keyboard', false);
+  this._editorActions = this._injector.get('editorActions', false);
+
+  if (this._keyboard) {
+    this._injector.invoke(KeyboardBindings, this);
```

Comment:
I think we're following the [create/append anything bindings here](https://github.com/bpmn-io/bpmn-js-create-append-anything/blob/main/lib/create-append-anything/keyboard-bindings/KeyboardBindings.js).

What I'd suggest is that we re-use that code to stay consistent.

Reply:
The create/append anything "appendElement" action triggers a contextPad entry which is no longer here because we replaced it with the append node. Since, we still want to re-use the "createElement" action, I removed the append action and added a new one that triggers the append node.

I'm happy to accept better suggestions but I'm not sure what you meant

Reply:
I think what is confusing about that particular line is that it is _conditionally_ creating the keyboard bindings _if_ the [keyboard is available](https://github.com/bpmn-io/diagram-js/blob/develop/lib/features/keyboard/KeyboardBindings.js#L40). Otherwise it is a noop.

Let's incorporate this as is and consider suggestions as a follow-up. I.e. we could make the dependency to `keyboard` explicit.